syntax = "proto3";
package outer.grpc;

import "outer/connector.proto";

option java_multiple_files = false;
option java_package = "com.cmcc.littlec.connector.service.outer";
/* option java_outer_classname = "Grpc"; */

// 通知服务，用于服务端推送通知至客户端
service ConnectorService {
	// 一对一服务请求
	rpc SendUnaryRequest(outer.connector.UnaryRequest) returns(outer.connector.UnaryResponse);
	// Session相关操作
	rpc SendSessionRequest(stream outer.connector.SessionRequest) returns(stream outer.connector.SessionNotify);
	// health检测
	rpc HealthCheck(outer.connector.HealthCheckRequest) returns(outer.connector.HealthCheckResponse);
}
